utils/dataloaders.py

```diff
@@ -672,7 +672,7 @@ def __getitem__(self, index):
         else:
             # Load image
             img, (h0, w0), (h, w) = self.load_image(index)
-
+            im0 = img.copy()
```

Comment:
if `im0` is just a copy of `img` which is a numpy array, why can't we use directly `img`?
Then we do not need to modify the dataloader